packages/ckeditor5-ui/src/menubar/utils.ts

```diff
@@ -56,24 +56,28 @@ export const MenuBarBehaviors = {
 	 */
 	toggleMenusAndFocusItemsOnHover( menuBarView: MenuBarView ): void {
 		menuBarView.on<MenuBarMenuMouseEnterEvent>( 'menu:mouseenter', evt => {
-			// This works only when the menu bar has already been open and the user hover over the menu bar.
-			if ( !menuBarView.isOpen ) {
+			// This behavior should be activated when one of condition is present:
+			// 1. The user opened any submenu of menubar and hover over items in the menu bar.
+			// 2. The user focused whole menubar using keyboard interaction and enabled focus borders and hover over items in the menu bar.
+			if ( !menuBarView.isFocusBorderEnabled && !menuBarView.isOpen ) {
```

Comment:
I don't see how these changes address the bug I was writing about so maybe the video will help.


https://github.com/ckeditor/ckeditor5/assets/1502228/ad477b03-5af0-47eb-bb12-5f12ffdfc734

At the beginning, I focus the editable, then I click Alt+F9. The menu bar gets focus. But after I move mouse around the menu bar, the focus is not moved. The blue outline does not follow the mouse and then I also click arrow keys, and they 1) don't start from where I have a mouse; 2) they don't trigger outline following the mouse.

Reply:
I am sorry - this was my mistake - I pulled wrong repo. It works fine, thanks.